sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterTraceExtensions.cs

```diff
@@ -31,6 +32,8 @@ public static TracerProviderBuilder AddAzureMonitorTraceExporter(this TracerProv
             // TODO: provide a way to turn off statsbeat
             // Statsbeat.InitializeAttachStatsbeat(options.ConnectionString);
 
+            builder.SetSampler(new ApplicationInsightsSampler(options.SamplingRatio));
```

Comment:
I think we should just document people to add the sampler themselves. Exporter should have no business setting sampler for user.